### PR TITLE
Release/5.1.0

### PR DESCRIPTION
**Pull request by ifrit98**
Minor update:
- Update blacklist and priority functions. THey now live in `_priority` and `_blacklist` https://github.com/opentensor/bittensor/commit/ae3b235652178e43ce97153144636d7f417a4788
- Add ABC for HuggingFace models (HuggingFaceMiner), lives in `_neuron/base_huggingface_miner.py` 
- Add several miners: `dolly-12b`, `dromedary-65b`, `fastchat_t5`, `guanaco`, `gpt4_x_vicuna`, `mpt_chat`, `raven` (RWKV), `wizard_vicuna`, etc https://github.com/opentensor/bittensor/commit/7aaa9611e8c47a03dfd6f51ea130f91cb80cc78f
- Add ABCs for all future miners/validators, lives in `_neuron/base_<miner|validator>.py` (setting us up for multi-modal subnets!)
- Add `is_set` for config defaults: https://github.com/opentensor/bittensor/commit/2407fb55e43467c5fbb5ccfe875dc7101658cfe9
- Minor fixes/updates to axon code: add subtensor param to `sync`
- Other small bug fixes/additions.

### CODE REVIEW

**Review by eduardogr — APPROVED** [review]
From my point of view, release-related things are OK,
I cannot talk related with the code itself, but from the release management point of view, looks good.

**Review by shibshib — APPROVED** [review]

File: bittensor/_blacklist/__init__.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+# THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+# THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+import argparse
+import bittensor
+from typing import Union, Tuple
+
+class blacklist:
```
Comment (Eugene-hu) [inline bittensor/_blacklist/__init__.py:22]: Are these classes used? or is this the future miners?

**Review by Eugene-hu — APPROVED** [review]
LGTM; are all of the prs into the release?

File: bittensor/_blacklist/__init__.py (hunk 1 above)
Comment (ifrit98) [reply bittensor/_blacklist/__init__.py:22]: This is a good catch, I need to integrate this into the current base_prompting_miner.py.  It's in https://github.com/opentensor/bittensor/blob/release/5.1.0/bittensor/_neuron/base_miner_neuron.py#L109 but those aren't used yet and will also be for future miners/validators.  

Will integrate tomorrow morning and test it out.

**Pull request merged**